packages/react-components/react-tags/stories/TagGroup/TagGroupSizes.stories.tsx

```diff
@@ -38,7 +42,7 @@ Sizes.storyName = 'Sizes';
 Sizes.parameters = {
   docs: {
     description: {
-      story: 'A TagGroup supports different sizes',
+      story: 'A TagGroup supports `medium`, `small` and `extra-small` size. Default size is `medium`.',
```

Comment:
I would mention that `size` on the `TagGroup` would set the default size for all its tags